recipes/guile/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --prefix="${PREFIX}"
+make -j 2
+make install
```

Comment:
Is there a `make check` we can run here?

Reply:
Mac fails on 1 test:
```
FAIL: threads.test: mutex-ownership: mutex with owner not retained (bug #27450)
```
The [referenced bug](http://savannah.gnu.org/bugs/?27450) was fixed in 2009.

Linux hangs while running srfi-18.test, which tests the thread library.  What's the best way to proceed?